code/src/components/TodoSummary.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { RemoveAll } from 'components/RemoveAll'
+import { RemoveDone } from 'components/RemoveDone'
+
+export const TodoSummary = () => {
+
+  const list = useSelector(store => store.todos.list)
```

Comment:
You're only using the items-array, so get only that instead:
`const items = useSelector(store => store.todos.list.items)`